mysql-test/suite/rpl/t/rpl_rewrite_db_sys_vars.opt

```diff
@@ -0,0 +1,2 @@
+"--replicate-rewrite-db=primary_db1->replica_db1"
+"--replicate-rewrite-db=x->y"
```

Comment:
This option file should only be used by the slave, so it should be renamed `rpl_rewrite_db_sys_vars-slave.opt`. Also a new line should be added to the end of the file.